### ISSUE TITLE
Difficulty calculator is inaccurate

### ISSUE THREAD

**Issue by swoolcock** [swoolcock/rush#28]

**Comment by frenzibyte** [comment]
I believe this is tracking the rush! difficulty calculator actually doing proper difficulty calculation, so https://github.com/swoolcock/rush/pull/124 won't close it yet, right?

**Comment by LumpBloom7** [comment]
No, it only serves as an interim solution, while that is absent.
